src/oss/confluent.sh

```diff
@@ -1433,6 +1479,65 @@ version_command() {
     fi
 }
 
+produce_usage() {
+    local exit_status="${1}"
+    if [[ -z "${exit_status}" ]]; then
+      exit_status=0
+    fi
+
+    cat <<EOF
+Usage: ${command_name} <topicname> [--value-format avro] [other optional args]
+
+Description:
+    Produce to Kafka topic specified by <topicname>.
+
+    Without '--value-format avro', it calls the 'kafka-console-producer' command,
```

Comment:
The mention to `kafka-console-produce` sounds like implementation detail that can be skipped. We shouldn't commit to that, it might change in the future. 

As above, I suggest we list only the intended allowed properties and we mention that localhost is used.